### PR TITLE
Create div-case-orchestration-service.json

### PR DESCRIPTION
**Pull request by njrich28**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: terraform-infra-approvals/div-case-orchestration-service.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "resources": [
+  ],
+  "module_calls": [
+
+    {"source":  "git@github.com:hmcts/cnp-module-webapp?ref=revert-159-tf-0.12.x"}
```
Comment (timja) [inline terraform-infra-approvals/div-case-orchestration-service.json]: why do we need this?

Comment (njrich28) [reply terraform-infra-approvals/div-case-orchestration-service.json]: testing something on div demo - doesn't apply the app service on clean build

Comment (timja) [reply terraform-infra-approvals/div-case-orchestration-service.json]: state file issue?

Comment (njrich28) [reply terraform-infra-approvals/div-case-orchestration-service.json]: was originally, but removed remote state file and all infra - from clean infra/state it didn't attempt to build the app service when `enable_ase = true`

**Pull request merged**